apps/web/src/components/FeatureFlag.tsx

```diff
@@ -200,7 +200,7 @@ export function FeatureFlag({
 
   const currentFlagValue = flag.values.find((f) => f.id === flagValueId)?.value;
 
-  if (!currentFlagValue) {
+  if (currentFlagValue === undefined) {
```

Comment:
couldn't `undefined` be a valid value? why not?

Reply:
We don't allow undefined as a value right now.

It's either an empty string, `false`, the integer `0` or an empty json object `{}`

Reply:
but `null` is okay?
are we checking here the result of the "find" function or could the actual `value` be undefined/null? if so, maybe split the check into two?

Reply:
you're right again, thanks :)